source/_drafts/Video-js-6-0-0-RC-0-The-first-RC.md

```diff
@@ -0,0 +1,24 @@
+---
+title: 'Video.js 6.0.0-RC.0: The first Release Candidate'
+author:
+  name: Gary Katsevman
+  github: gkatsev
+tags:
+  - video.js 6.0
+---
+
+## Release Candiate Released
+Last week, we published the first Release Candidate (RC) of Video.js 6.0 on npm.
+This was a months-long team effort to make Video.js even better.
```

Comment:
I think that this could be combined with the sentence above to make it more readable. Something like: 
"Last week, we started to wrap up months of effort by publishing the first Release Candidate (RC) of Video.js 6."